homeassistant/components/thermostat/__init__.py

```diff
@@ -301,22 +306,32 @@ def turn_fan_off(self):
     @property
     def min_temp(self):
         """Return the minimum temperature."""
-        return convert(7, TEMP_CELSIUS, self.unit_of_measurement)
+        unit = self.unit_of_measurement
+        if not unit:
```

Comment:
Always pull from config. All entities will have same unit because we
convert

On Thu, Aug 4, 2016, 09:10 Teagan Glenn <notifications@github.com> wrote:

> In homeassistant/components/thermostat/__init__.py
> <https://github.com/home-assistant/home-assistant/pull/2705#discussion_r73552815>
> :
>
> > @@ -301,22 +306,32 @@ def turn_fan_off(self):
> >      @property
> >      def min_temp(self):
> >          """Return the minimum temperature."""
> > -        return convert(7, TEMP_CELSIUS, self.unit_of_measurement)
> > +        unit = self.unit_of_measurement
> > +        if not unit:
>
> So - this basically wipes out the heat control component. It has literally
> no way of setting the units; I'm pretty sure it was just grabbing the unit
> set for HASS. Options:
>
> Pull unit from hass config
> Add attribute to device configuration
> Both
>
> I assume we should do both - just want input on it.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/home-assistant/home-assistant/pull/2705/files/dcac2dd629869be07e05bd8e4d7ac96956517873#r73552815>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABYJ2iuQEpYmMmhDyteo-Gld--UHesEwks5qcg7wgaJpZM4JaO1y>
> .
>